templates/vue/cypress.gha.json

```diff
@@ -3,7 +3,8 @@
     "ADMIN_USERNAME": "admin",
     "ADMIN_PASSWORD": "secret",
     "SUBSCRIBER_USERNAME": "subscriber",
-    "SUBSCRIBER_PASSWORD": "subscriber"
+    "SUBSCRIBER_PASSWORD": "subscriber",
+    "TAPESTRY_ENV": "TYDE"
```

Comment:
Not sure we need this. When would this not be `TYDE`?

Reply:
Good point. I guess in this branch, it would always be `TYDE`. I'll remove it and update the conditional mentioned above.